templates/android/docs/java/example.md.twig

```diff
@@ -1,93 +1,67 @@
-import androidx.appcompat.app.AppCompatActivity
-import android.os.Bundle
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.launch
+{% import 'kotlin/base/utils.twig' as utils %}
 import {{ sdk.namespace | caseDot }}.Client
 {% if method.parameters.all | filter((param) => param.type == 'file') | length > 0 %}
 import {{ sdk.namespace | caseDot }}.models.InputFile
 {% endif %}
 import {{ sdk.namespace | caseDot }}.services.{{ service.name | caseUcfirst }}
 
-public class MainActivity extends AppCompatActivity {
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-
-        Client client = new Client(getApplicationContext())
+Client client = Client(context)
```

Comment:
Why did we decide to remove all the android reference or something?

Reply:
It seems redundant to have a lot of extra code that is not related/helpful to the example